PokeDemo/CustomViews/TableViewCells/PokemonListCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PokemonListCell.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import UIKit
+
+class PokemonListCell: BaseTableViewCell {
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var title: UILabel!
+    @IBOutlet weak var icon: UIImageView!
+    
+    // MARK: - Exposed Methods
+
+    func setupData(data: Pokemon) {
+        title.text = data.name?.uppercased()
```

Comment:
not sure what happens when name is nil. What will it show as title?

Reply:
It will show blank in case name is nil